Dockerfile

```diff
@@ -4,15 +4,15 @@ ARG TARGET=server
 ARG GOPROXY
 
 # Build tcheck binary
-FROM golang:1.17-alpine3.15 AS tcheck
+FROM golang:1.17.13-alpine3.15 AS tcheck
```

Comment:
Update this anyway so that both share the same version of Golang even though it isn't vulnerable.